### DISCUSSION TITLE
Adding "iss" to JWT returned by Supabase

### DISCUSSION THREAD

**Discussion by mmalesadev** [supabase/supabase#13536] [Questions] [answered]
Hi,
I need the JWT returned by Supabase after successful authentication to have the "iss" claim, which it doesn't have.
Can it be easily done somehow? This is needed by the Kong Gateway's jwt plugin.

**Comment by GaryAustin1** [comment, marked as answer]
There is no ability to add to the jwt claims from supabase gotrue.
You would need to modify the jwt on a server/edge function using a jwt library and the jwt secret key.
Otherwise you can submit an enhancement request on the supabase/gotrue.
